### PR TITLE
Shooter

### PR DESCRIPTION
**Pull request by magutheragu**
Shooter :) ☄️
Both motors controlled by one trigger.

### CODE REVIEW

**Review by End10 — APPROVED** [review]

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -16,9 +16,14 @@
 import frc.robot.Constants.DriveConstants.joysticks;
 import frc.robot.auto.*;
 import frc.robot.commands.Hanger.*;
+import frc.robot.commands.Shooter.ShooterForwardCmd;
+import frc.robot.commands.Shooter.ShooterStopCmd;
 import frc.robot.commands.TeleopCmd;
 import frc.robot.subsystems.DrivetrainSubsystem;
 import frc.robot.subsystems.HangerSubsystem;
+import frc.robot.subsystems.ShooterSubsystem;
+import frc.robot.commands.Hanger.*;
```
Comment (langton) [inline src/main/java/frc/robot/RobotContainer.java]: Duplicate import?

Comment (magutheragu) [reply src/main/java/frc/robot/RobotContainer.java]: fixed? maybe?

**Review by madelinec11 — APPROVED** [review]
all gucci ;)

**Review by mastermind-mayhem — APPROVED** [review]

**Review by mastermind-mayhem — APPROVED** [review]

**Comment by magutheragu** [comment]
FINALLy

**Comment by magutheragu** [comment]
YIPEEEEEEEEEEEE

**Pull request merged**